src/main/scala/com/yotpo/metorikku/metric/Metric.scala

```diff
@@ -117,28 +125,50 @@ case class Metric(configuration: Configuration, metricDir: File, metricName: Str
     }
   }
 
+  private def reportLagTime(dataFrame: DataFrame, reportLagTimeColumn: Option[String], instrumentationProvider: InstrumentationProvider) ={
+    reportLagTimeColumn match {
+      case Some(timeColumn) => {
```

Comment:
Can you please add a case when timeColumn is not defined and throw an exception?

Reply:
Regarding influx test, when comparing results the current timestamp will never be as System.currentTimeMillis , wdyt?